shopelectro/tests/tests_selenium.py

```diff
@@ -32,6 +32,12 @@ def is_cart_empty(browser):
     return browser.find_element_by_class_name('js-cart-is-empty').is_displayed()
 
 
+def cart_total(browser):
```

Comment:
let's subtask here selenium.elements.Cart creating. It'll responsible on the cart block at the top right corner